skt/reporter.py

```diff
@@ -436,21 +436,23 @@ def getreport(self):
         return '\n'.join(msg)
 
     def getsubject(self):
-        subject = "[skt] [%s] " % ("PASS"
-                                   if self.cfg.get("retcode") == "0"
-                                   else "FAIL")
-
-        if self.mergedata.get("base"):
-            subject += "[%s] " % self.mergedata['base'][0].split("/")[-1]
+        subject = '[skt] %s: ' % self.mergedata.get('base')[0].split("/")[-1]
 
         if self.cfg.get("mergelog"):
-            subject += "patch application failed"
+            subject += "Patch application failed"
         elif self.cfg.get("buildlog"):
-            subject += "build failed"
+            subject += "Build failed"
         else:
-            subject += "result report"
-            if self.cfg.get("krelease"):
-                subject += " for kernel %s" % self.cfg.get("krelease")
+            subject += "Report"
+
+        if self.cfg.get("krelease"):
+            subject += " for kernel %s" % self.cfg.get("krelease")
+
+        if not self.cfg.get('mergelog') and not self.cfg.get('buildlog'):
+            if self.cfg.get('retcode') == '0':
+                subject += ': PASS'
+            else:
+                subject += ': FAIL'
```

Comment:
Hmm, I liked the PASS/FAIL at the start of the subject. I think it's the most important piece of information and should be at the start. I would even vote for removing `[skt]`. Also, I don't think the git repo name is that important. How often do people submit patches to different repos/maillists at the same time, anyway?

What do you think?